docs/Installation-Guide.rst

```diff
@@ -0,0 +1,303 @@
+Installation Guide
+==================
+
+Here is the guide for the build of CLI version.
+
+For the build of Python-package and R-package, please refer to `Python-package`_ and `R-package`_ folders respectively.
+
+Windows
+~~~~~~~
+
+LightGBM can use Visual Studio, MSBuild with CMake or MinGW to build in Windows.
+
+Visual Studio (or MSBuild)
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With GUI
+********
+
+1. Install `Visual Studio`_.
+
+2. Download `zip archive`_ and unzip it.
+
+3. Go to ``LightGBM-master/windows`` folder.
+
+4. Open ``LightGBM.sln`` file with Visual Studio, choose ``Release`` configuration and click ``BUILD-> Build Solution (Ctrl+Shift+B)``.
```

Comment:
need to select ```X64``` as well ?

Reply:
There is no another choice neither at start windows, nor in project settings:

![image](https://user-images.githubusercontent.com/25141164/31308871-49b89368-ab86-11e7-9ce8-a8d0ece6c0d1.png)
![image](https://user-images.githubusercontent.com/25141164/31308876-62e881a4-ab86-11e7-96c4-d4ba2c8d6c0a.png)